pkgs/applications/version-management/yadm/default.nix

```diff
@@ -1,18 +1,43 @@
-{ lib, stdenv, fetchFromGitHub, git, gnupg, installShellFiles }:
+{ lib
+, stdenv
+, resholvePackage
+, fetchFromGitHub
+, git
+, bash
+, openssl
+, gawk
+/*
+TODO: yadm can use git-crypt and transcrypt
+but it does so in a way that resholve 0.6.0
+can't yet do anything smart about. It looks
+like these are for interactive use, so the
+main impact should just be that users still
+need both of these packages in their profile
+to support their use in yadm.
+*/
+# , git-crypt
+# , transcrypt
+, j2cli
+, esh
+, gnupg
+, coreutils
+, gnutar
+, installShellFiles
+, runCommand
+, yadm
+}:
 
-stdenv.mkDerivation rec {
+resholvePackage rec {
   pname = "yadm";
   version = "3.1.1";
 
-  buildInputs = [ git gnupg ];
-
   nativeBuildInputs = [ installShellFiles ];
 
   src = fetchFromGitHub {
     owner  = "TheLocehiliosan";
     repo   = "yadm";
     rev    = version;
-    sha256 = "sha256-bgiRBlqEjDq0gQ0+aUWpFDeE2piFX3Gy2gEAXgChAOk=";
+    hash   = "sha256-bgiRBlqEjDq0gQ0+aUWpFDeE2piFX3Gy2gEAXgChAOk=";
```

Comment:
not sure why you changed that from sha256 to hash

Reply:
Now that you bring it up, I'm not really certain about this. 

I've inferred from the introduction of SRI hashes and their use in the CLI toolchain that the long-term goal was transitioning from `sha256 = "<hash>"` to `hash = "<algo>-hash"`, so I generally do these whenever I'm already touching a hash and ignore them otherwise.

This specific case is an outlier because I actually converted this back before an r-ryantm bump in https://github.com/NixOS/nixpkgs/commit/36e1c8753ceabca303fbc0d9600663afbb22957f which left the attr as sha256 but updated the format to SRI. using both the sha256 attr and SRI seemed redundant to me, so I kept the name change when I resolved conflicts even though I wasn't otherwise touching it.